### PR TITLE
Merge dev branch

### PR DESCRIPTION
**Pull request by XhmikosR**
Only issue left is the HTML validation errors.

/CC @vladikoff

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-sudo: false
+sudo: required
+dist: trusty
 language: node_js
 node_js:
   - "0.10"
   - "0.12"
```
Comment (vladikoff) [inline .travis.yml]: we can probably make the website node 4 +

File: package.json [hunk 2]
```diff
@@ -19,34 +19,35 @@
...
-    "grunt-cli": "~0.1.x",
-    "grunt-contrib-clean": "~0.5.0",
-    "grunt-contrib-copy": "~0.4.1",
-    "grunt-contrib-less": "~0.5.2",
-    "grunt-contrib-uglify": "~0.3.3",
-    "grunt-docs": "git://github.com/gruntjs/grunt-docs#a9b78876082155691fc2728b6ed28b60bc534a5c",
-    "highlight.js": "~7.3.0",
-    "jade": "~0.35.0",
-    "lodash": "~1.0.1",
+    "async": "^2.0.0",
```
Comment (vladikoff) [inline package.json]: I'd rather keep ~

File: .travis.yml (hunk 1 above)
Comment (XhmikosR) [inline .travis.yml]: OK, agreed.

**Comment by vladikoff** [comment]
@XhmikosR Amazing, thanks so much!! 👍

**Pull request merged**